sklearn/ensemble/__init__.py

```diff
@@ -17,6 +17,8 @@
 from .gradient_boosting import GradientBoostingClassifier
 from .gradient_boosting import GradientBoostingRegressor
 from .voting_classifier import VotingClassifier
+from .._fast_gradient_boosting import FastGradientBoostingClassifier
+from .._fast_gradient_boosting import FastGradientBoostingRegressor
```

Comment:
why should this not live under ensemble?

Reply:
The code is in sklearn/_fast_gradient_boosting and the estimators are exposed in sklearn.ensemble. For some reason, I wasn't able to put the code in a subdirectory of sklearn/ensemble because of some weird Cython bug (couldn't find a pxd file). I might have done something wrong so if keeping sklearn/_fast_gradient_boosting isn't great I can take another look.